src/components/generics/displayWrapper.js

```diff
@@ -17,7 +17,6 @@ export const DisplayWrapper = styled("div")(
         display: "flex",
         flexDirection: "row",
         flexWrap: "wrap",
-        justifyContent: "space-between",
```

Comment:
why was this line removed? I feel like removing it could be problematic as some of our flex rows assume it is space-between

Reply:
I removed it because space between was not always the desired behavior in every flexRow. For instance in the footer I actually don't want space between to be active on the indexes. I added justifyContent: space-between separately in all areas that previously needed that behavior so everything should still look correct.